name.abuchen.portfolio/src/name/abuchen/portfolio/money/currencies_de.properties

```diff
@@ -337,6 +337,16 @@ VUV = Vatu
 
 WST = Samoanischer Tala
 
+XAF = Zentralafrikanischer CFA-Franc
+
+XCD = Ostkaribischer Dollar
+
+XDR = IWF Sonderziehnungsrechte
```

Comment:
1) Should have a hyphen instead of a space. 2) Has an additional n (Zieh**n**ung).

Reply:
Thank you. I fixed it.